src/parser.rs

```diff
@@ -233,9 +241,10 @@ impl FromStr for CypherValue {
 impl Display for CypherValue {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            CypherValue::Float(float) => write!(f, "{}", float),
-            CypherValue::Integer(integer) => write!(f, "{}", integer),
-            CypherValue::String(string) => f.pad(string),
+            CypherValue::Float(float)       => write!(f, "{}", float),
+            CypherValue::Integer(integer)   => write!(f, "{}", integer),
+            CypherValue::String(string)     => f.pad(string),
+            CypherValue::Boolean(boolean)   => write!(f, "{}", boolean),
```

Comment:
nice alignment on the indentation! is that what `cargo fmt` produces?

Reply:
No, it was made manually.